oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -96,7 +96,8 @@ public class RevisionsCommand implements Command {
     private static final List<String> LOGGER_NAMES = of(
             "org.apache.jackrabbit.oak.plugins.document.VersionGarbageCollector",
             "org.apache.jackrabbit.oak.plugins.document.NodeDocumentSweeper",
-            "org.apache.jackrabbit.oak.plugins.document.VersionGarbageCollector.detailedGC"
+            "org.apache.jackrabbit.oak.plugins.document.VersionGarbageCollector.auditDGC",
+            "org.apache.jackrabbit.oak.spi.state.AbstractNodeState"
```

Comment:
is this still needed with he https://github.com/apache/jackrabbit-oak/pull/1417/commits/75eec42dfe4030de30254da688ddb9cbc60c57d7 revert?

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1417/commits/d4406db8cfcf27f5197638d235cdeb216944a4f6